### ISSUE TITLE
Is this project dead? How can we help?

### ISSUE THREAD

**Issue by BretFisher** [samneirinck/posh-docker#26]
Hey @samneirinck, I know a lot of docker users that would love this updated to current CLI commands and sub-commands, options, etc.

Maybe we Docker Captains can help spread the word and get you some help with fixes and new features, but I have two questions first:

1. Are you still willing to lead this repo by accepting PR's, commenting on issues, etc? It would be great if we could use the same PowerShell Module name and repo.

2. Can you describe a list of issues or needed features to get it working with old and new command format and options?

CC: @StefanScherer

**Comment by StefanScherer** [comment]
Thanks Captain @BretFisher ;-)
I encourage my team to use WSL right now, but for the community the posh-docker is still very useful to have same tab expansion experience like in bash.

**Comment by samneirinck** [comment]
Thanks for bringing this to my attention. It's true the project has been pretty silent for quite a while, mainly due to the fact I've switched to bash+WSL to interact with Docker/Kubernetes.

I've tried a few times to introduce support for the management commands, however the implementation was not so trivial, based on the regex approach I used for the tab completion. Other tab completions for docker, like bash, fish, ... use a more extensive system, pretty much handling each command separately. This allows them to clearly identify what should be completed in any scenario. I've done some initial prototyping on this in the [pester-tests branch](https://github.com/samneirinck/posh-docker/tree/pester-tests).

To answer your 2 questions:
1. I can accept PR's, comment on issues, etc, but I'm also willing to give some other people the permissions to do the same. Since I'm no longer using it myself, the project is probably better off with some active maintainers, who are dedicated to support the latest features or bug fixes. 
2. With regards to a list of issues, I think the main issue is to support management commands. However, as noted before, it might involve some serious rework of the module (no longer based on only regex).

Let me know what you all think would be the best way to proceed.

**Comment by matt9ucci** [comment]
@BretFisher @StefanScherer @samneirinck
Here's my completion module ["DockerCompletion"](https://www.powershellgallery.com/packages/DockerCompletion) working for the management commands. It is customizable and not based on the regex approach (See [this](https://github.com/matt9ucci/DockerCompletion/blob/master/DockerCompletion/completers.ps1) and [this](https://github.com/matt9ucci/DockerCompletion/blob/master/DockerCompletion/completers4arguments.ps1)).

Note: It requires PowerShell >= 5.0 and docker cli >= 1.13

**Comment by BretFisher** [comment]
@matt9ucci This looks great, do you consider it a replacement for posh-docker? (superset). It looks like you've had it awhile, I'll give it a shot!

**Comment by matt9ucci** [comment]
@BretFisher Yes. The DockerCompletion works roughly the same as the bash completion for docker. I have started to develop it last year. The latest version supports Docker v18.03.0. The next release will support v18.06.0. Feedback is welcomed!

I think posh-docker is good for the users who use PowerShell v4.0 or lower and Docker v1.12 or lower. But for the users who use PowerShell v5.0+, including PowerShell Core, and Docker v1.13+, the DockerCompletion should be useful.

**Comment by samneirinck** [comment]
Perhaps we can just add a note to the READNE here then to point towards @matt9ucci's excellent work?

**Comment by samneirinck** [comment]
I've updated the README to clarify the current status, and point to the DockerCompletion github page.
